code/modules/urist/items/clothes/head.dm

```diff
@@ -318,6 +318,109 @@ obj/item/clothing/head/princessbow
 	cold_protection = HEAD
 	min_cold_protection_temperature = 243.15
 
+
+
+//Civil War hats
+
+/obj/item/clothing/head/urist/USLTGenHat
```

Comment:
can you change the path on these to /obj/item/clothing/head/urist/civilwar/USLTGenHat and etc. It makes the object tree cleaner. Also, don't use CamelCase in paths, just make them all lower case.